recipes/libcurl/all/conanfile.py

```diff
@@ -213,13 +210,30 @@ def build_requirements(self):
         else:
             self.tool_requires("libtool/2.4.6")
             self.tool_requires("pkgconf/1.7.4")
-            if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
+            if self._settings_build.os == "Windows" and not get_env("CONAN_BASH_PATH") and not self.conf.get("tools.microsoft.bash:path", default=False, check_type=bool):
                 self.tool_requires("msys2/cci.latest")
 
+    def layout(self):
+        if self._is_using_cmake_build:
+            cmake_layout(self, src_folder="src")
+        else:
+            basic_layout(self, src_folder="src")
+
     def source(self):
         get(self, **self.conan_data["sources"][self.version],
-                  destination=self._source_subfolder, strip_root=True)
-        download(self, "https://curl.haxx.se/ca/cacert.pem", "cacert.pem", verify=True)
+                  destination=self.source_folder, strip_root=True)
+        download(self, "https://curl.haxx.se/ca/cacert.pem", "cacert.pem", verify=True, sha256="2cff03f9efdaf52626bd1b451d700605dc1ea000c5da56bd0fc59f8f43071040")
```

Comment:
I just had to update the sha256 that changed. Shouldn't we remove it to prevent build failure ?

Reply:
I don't think that would be safe. Is there a way to reference a download that won't be updated?

Reply:
Please, take a look on ApprovalTest conandata.yml as reference to download that license file: https://github.com/conan-io/conan-center-index/blob/master/recipes/approvaltests.cpp/all/conandata.yml

Reply:
The ApprovalTest example doesn't has to mix between tar.gz and pem files.